lib/Remote/Lab.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+=head1 Lab
+
+=head1 SYNOPSIS
+
+Setup VPN to a remote lab using openconnect compatible with Cisco AnyConnect
+VPN and connect.
+
+=cut
+
+package Remote::Lab;
+use strict;
+use warnings;
+use base 'Exporter';
+use Exporter;
+use registration 'add_suseconnect_product';
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+
+our @EXPORT = qw(setup_vpn connect_vpn setup_ssh_tunnels);
+
+
+sub setup_vpn {
+    my ($self) = @_;
+    add_suseconnect_product('PackageHub', undef, undef, undef, 300, 1) if is_sle;
+    zypper_call 'in --no-recommends openconnect';
+    script_run 'read -s vpn_password', 0;
```

Comment:
interesting way to minimize password availability in openQA open test data. I vote for extracting it into separate function which can be used not just in this test. But it can be done in separate PR and not obligatory by you ;)

Reply:
yes, my plan is to use the same approach in a broader context after this PR is merged.